### ISSUE TITLE
zellij doesn't show anything

### ISSUE THREAD

**Issue by AlexLurker** [zellij-org/zellij#2497]
zellij start up but doesn't show anything  

```
% zellij --version
zellij 0.36.0
```

```
% stty size       
63 238
```

```
% uname -av
FreeBSD 13.2-RELEASE FreeBSD 13.2-RELEASE releng/13.2-n254617-525ecfdad597 GENERIC amd64
```

```
cat /tmp/zellij-2001/zellij-log/zellij.log 
INFO   |zellij_client            | 2023-05-30 22:57:08.598 [main      ] [zellij-client/src/lib.rs:143]: Starting Zellij client! 
INFO   |zellij_server            | 2023-05-30 22:57:08.646 [main      ] [zellij-server/src/lib.rs:237]: Starting Zellij server! 
ERROR  |zellij_utils::errors::not| 2023-05-30 22:57:08.649 [server_listener] [zellij-utils/src/errors.rs:585]: Panic occured:
             thread: server_listener
             location: At zellij-server/src/lib.rs:281:55
             message: called `Result::unwrap()` on an `Err` value: Os { code: 79, kind: Uncategorized, message: "Inappropriate file type or format" } 

```

**Comment by tlinford** [comment]
This has been fixed in main by #2424 but it's not yet released. v0.35.2 should not have this problem.

**Comment by AlexLurker** [comment]
> This has been fixed in main by #2424 but it's not yet released. v0.35.2 should not have this problem.

I'll wait for the new version and close this issue =)
